aii-opennebula/src/main/perl/opennebula.pm

```diff
@@ -496,7 +503,7 @@ sub remove_and_create_vm_template
             $main::this_app->info("No QUATTOR flag found for VM template: ",$t->name);
         }
     }
-    
+
     if ($createvmtemplate && !$remove) {
```

Comment:
I don't understand how this works, but probably  I don't completely get what the `$remove` flag means